server/web_templates.go

```diff
@@ -90,7 +92,7 @@ var indexTemplate = template.Must(template.New("index.html.tmpl").Parse(`
     {{ range .Locks }}
       <a href="{{ $basePath }}{{.LockPath}}">
         <div class="twelve columns button content lock-row">
-        <div class="list-title">{{.RepoFullName}} - <span class="heading-font-size">#{{.PullNum}}</span></div>
+        <div class="list-title">{{.RepoFullName}} <span class="heading-font-size">#{{.PullNum}}</span> - <small>dir:<code>{{.Path}}</code> workspace:<code>{{.Workspace}}</code></small></div>
```

Comment:
```suggestion
        <div class="list-title">{{.RepoFullName}} <span class="heading-font-size">#{{.PullNum}}</span> - <code>{{.Path}}</code> <code>{{.Workspace}}</code></div>
```

I'm worried this is too small:
![image](https://user-images.githubusercontent.com/1034429/64900373-a6bcf680-d645-11e9-806f-ec0f8d8c1274.png)

What do you think about:
![image](https://user-images.githubusercontent.com/1034429/64900379-b6d4d600-d645-11e9-9e10-a31e27029eb7.png)
People will probably know that it's a dir and a workspace.